ztp/source-crs/ClusterVersionUpdateGraph.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.openshift.io/v1
```

Comment:
Hmm, why do we need 2 files for the same CR?

Reply:
This provides us the ability to workaround the CVO issue. This CR can be used to update channel/upstream first, and the normal one above to be used for upgrade. After the fix of the CVO bz is back-ported to 4.9, we can safely delete this CR without impact.

Reply:
we are trying to separate updating the channel and version. That is why we are creating 2 different CRs.

Reply:
If this were the only CR, could the user accomplish the same thing through overlay with the PGT? They already need to supply the status section above in order to fill in the version.

Reply:
Agreed with one CR, and this CR should have channel and upstream explicitly specified. This will help with image precaching of ocp image, which expects channel, upstream and desiredUpdate to be found in one policy.